common/tools/dev-tool/src/commands/run/vendored.ts

```diff
@@ -54,7 +54,7 @@ export const commandInfo = makeCommandInfo("vendored", "run dev-tool's dependenc
 
 export default async (...args: string[]): Promise<boolean> => {
   // I'm not 100% sure what underscore-prefixed commands do, but the only one we have now is _mocha.
```

Comment:
maybe we should create another issue to remove this handling since we've moved away from mocha?

Reply:
Logged https://github.com/Azure/azure-sdk-for-js/issues/36252